vector/src/main/res/values/strings.xml

```diff
@@ -2177,7 +2177,7 @@
     <string name="direct_room_profile_section_more_leave">Leave</string>
     <string name="room_profile_leaving_room">"Leaving the room…"</string>
 
-    <string name="room_member_override_nick_color">Override nick color</string>
+    <string name="room_member_override_display_name_colour">Override display name colour</string>
```

Comment:
Following @fedrunov remark, the result will be:

```suggestion
    <string name="room_member_override_nick_color">Override display name color</string>
```

(default resource are in US English, so we use `color`)

Thanks